input/fsh/au-erequesting-diagnosticrequest.fsh

```diff
@@ -1,7 +1,7 @@
-Profile: AUeRequestingServiceRequest
+Profile: AUeRequestingDiagnosticRequest
```

Comment:
Commenting here to get an inline comment. 
Please update the example in ['Example: AU eRequesting ServiceRequest profile showing identifier labelled Must Support' in the Conformance page. ](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/master/conformance.html#presentation-of-elements-labelled-must-support-in-raw-representations)

Reply:
Amended, thanks.

Reply:
Please updated profile title in the example caption as well.

Reply:
Amended, thanks.